### PR TITLE
refine transforms_tests/bbox_tests

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

File: tests/transforms_tests/bbox_tests/test_flip_bbox.py [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 from chainercv.transforms import flip_bbox
+from chainercv.utils.testing.generate_random_bbox import generate_random_bbox
 
 
 class TestFlipBbox(unittest.TestCase):
 
     def test_flip_bbox(self):
-        bbox = np.random.uniform(
-            low=0., high=32., size=(10, 4))
+        bbox = generate_random_bbox(10, (32, 32), 0, 32)
```
Comment (yuyu2172) [inline tests/transforms_tests/bbox_tests/test_flip_bbox.py]: How about using non-square images.

File: tests/transforms_tests/bbox_tests/test_resize_bbox.py [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
 from chainercv.transforms import resize_bbox
+from chainercv.utils.testing.generate_random_bbox import generate_random_bbox
 
 
 class TestResizeBbox(unittest.TestCase):
 
     def test_resize_bbox(self):
-        bbox = np.random.uniform(
-            low=0., high=32., size=(10, 4))
+        bbox = generate_random_bbox(10, (32, 32), 0, 32)
```
Comment (yuyu2172) [inline tests/transforms_tests/bbox_tests/test_resize_bbox.py]: ditto

File: tests/transforms_tests/bbox_tests/test_translate_bbox.py [hunk 3]
```diff
@@ -4,13 +4,13 @@
...
 from chainercv.transforms import translate_bbox
+from chainercv.utils.testing.generate_random_bbox import generate_random_bbox
 
 
 class TestTranslateBbox(unittest.TestCase):
 
     def test_translate_bbox(self):
-        bbox = np.random.uniform(
-            low=0., high=32., size=(10, 4))
+        bbox = generate_random_bbox(10, (32, 32), 0, 32)
```
Comment (yuyu2172) [inline tests/transforms_tests/bbox_tests/test_translate_bbox.py]: ditto

**Comment by knorth55** [comment]
Updated

**Pull request merged**